app/code/Magento/LoginAsCustomerAdminUi/README.md

```diff
@@ -1,3 +1,38 @@
-# Magento_LoginAsCustomerSales module
+# Magento_LoginAsCustomerAdminUi module
 
-The Magento_LoginAsCustomerAdminUi module provides UI for Admin Panel
+This module provides UI for Admin Panel for Login As Customer functionality.
+
+## Installation
+
+For information about a module installation in Magento 2, see [Enable or disable modules](https://devdocs.magento.com/guides/v2.4/install-gde/install/cli/install-cli-subcommands-enable.html).
+
+## Extensibility
+
+Extension developers can interact with the Magento_LoginAsCustomerAdminUi module. For more information about the Magento extension mechanism, see [Magento plugins](http://devdocs.magento.com/guides/v2.4/extension-dev-guide/plugins.html).
+
+[The Magento dependency injection mechanism](http://devdocs.magento.com/guides/v2.4/extension-dev-guide/depend-inj.html) enables you to override the functionality of the Magento_LoginAsCustomerAdminUi module.
+
+### Layouts
+
+This module introduces the following layouts in the `view/adminhtml/layout` directory:
```

Comment:
This comment is still not fully resolved - there's the layouts vs layout handles thing. 
When handles are mentioned, it's about those xml files that define the entire page structure - something you can assign your new page and it will look properly, like `app/code/Magento/Theme/view/base/page_layout/empty.xml`:
```
<?xml version="1.0"?>
<!--
/**
 * Copyright © Magento, Inc. All rights reserved.
 * See COPYING.txt for license details.
 */
-->
<layout xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_layout.xsd">
    <container name="root">
        <container name="after.body.start" as="after.body.start" before="-" label="Page Top"/>
        <container name="page.wrapper" as="page_wrapper" htmlTag="div" htmlClass="page-wrapper">
            <container name="global.notices" as="global_notices" before="-"/>
            <container name="main.content" htmlTag="main" htmlId="maincontent" htmlClass="page-main">
                <container name="columns.top" label="Before Main Columns"/>
                <container name="columns" htmlTag="div" htmlClass="columns">
                    <container name="main" label="Main Content Container" htmlTag="div" htmlClass="column main"/>
                </container>
            </container>
            <container name="page.bottom.container" as="page_bottom_container" label="Before Page Footer Container" after="main.content" htmlTag="div" htmlClass="page-bottom"/>
            <container name="before.body.end" as="before_body_end" after="-" label="Page Bottom"/>
        </container>
    </container>
</layout>
```
And you can use them in other layouts or layout handles like this:
```
<page layout="admin-1column" xmlns:xsi="..." xsi:noNamespaceSchemaLocation="...">
    <body>
        ...
```
Layouts are usually (not always - see `app/code/Magento/Robots/view/frontend/page_layout/robots.xml`) defined in `layouts.xml` files, like this:
```
<page_layouts xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/PageLayout/etc/layouts.xsd">
    <layout id="1column">
        <label translate="true">1 column</label>
    </layout>
    <layout id="2columns-left">
        <label translate="true">2 columns with left bar</label>
    </layout>
    <layout id="2columns-right">
        <label translate="true">2 columns with right bar</label>
    </layout>
    <layout id="3columns">
        <label translate="true">3 columns</label>
    </layout>
</page_layouts>
```
They reside in page_layout directories.

Layout handles, which live in layout directories, are similar, but they don't represent full page structure. They may rely on one, but their main function is add instructions that are specific to that handle. There are two types of those - handles that correspond to actions like `catalog_product_view.xml` (sometimes with additional parameters) and handles that don't do that and contain layout instructions that are shared by multiple other layout handles, which can include them like this:
```
<layout xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/layout_generic.xsd">
    <update handle="loginascustomer_confirmation_popup" />
</layout>
```

In this case, `loginascustomer_confirmation_popup.xml` is not a standalone top-to-bottom layout file. It's a handle. It's referenced by a few other handles in the same directory.